packages/open-api-gateway/scripts/generators/python/templates/operationConfig.mustache

```diff
@@ -188,10 +201,24 @@ def {{operationId}}_handler(_handler: {{operationIdCamelCase}}HandlerFunction =
                 context,
                 interceptor_context,
             ), **kwargs)
+
+            response_body = ''
+            if response.body is None:
+                pass
```

Comment:
previously we would return '' as part of the response. Does passing achieve the same result?

Reply:
Yep, `response_body` will remain unchanged and so be used as `''`